src/nft/components/collection/Sweep.tsx

```diff
@@ -177,13 +177,17 @@ export const Sweep = ({ contractAddress, minPrice, maxPrice }: SweepProps) => {
   const nft20Assets = useLazyLoadSweepAssetsQuery(nft20Params)
 
   const { sortedAssets, sortedAssetsTotalEth } = useMemo(() => {
-    if (!collectionAssets || !nftxAssets || !nft20Assets)
+    if (!collectionAssets && !nftxAssets && !nft20Assets) {
```

Comment:
why this change?

Reply:
if any markets were selected it would return undefined because nftxAssets or nft20 assets would be undefined